index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
+            based on the information from the processor’s
+            parameterDescriptors getter and initialize parameter values from
+            options where the keys match names declared in
+            AudioParamDescriptors.
+          </li>
+          <li>
+            <em>In the window scope</em>: Return from the AudioWorkletNode
+            constructor.
+          </li>
+        </ol>
+        <section>
+          <h2 id="AudioWorkletNode">
+            The <dfn>AudioWorkletNode</dfn> interface
+          </h2>
+          <p>
+            This interface represens an <a>AudioNode</a> which lives on the main
+            control thread. The user can create this node from an instance of
+            <a>AudioContext</a> and such node can be connected with other built-in
+            <a>AudioNode</a>s to form an audio graph.
+          </p>
+          <dl title=
+          "[Constructor(AudioContext context, DOMString name, optional AudioWorkletNodeOptions options)] interface AudioWorkletNode : AudioNode" 
+          class="idl">
+            <dt>
+              attribute EventHandler onmessage
+            </dt>
+            <dd>
+              <p>
+                The onmessage handler is called whenever the associated
+                <a>AudioWorkletProcessor</a> posts a message back to the main
+                thread.
+              </p>
+            </dd>
+            <dt>
+              void postMessage(any message, optional sequence&lt;Transferable&gt;
+              transfer)
+            </dt>
+            <dd>
+              <p>
+                postMessage May be called to send a message to the corresponding
+                <a>AudioWorkletProcessor</a> via the algorithm defined by the
+                Worker specification.
+              </p>
+            </dd>
+          </dl>
+
+          <section>
+            <h2 id="AudioWorkletNodeOptions">
+              AudioWorkletNodeOptions
+            </h2>
+            <p>
+              The AudioNodeOptions dictionary can be used for the custom
+              initialization of <a>AudioNode</a> attributes in the
+              <a>AudioWorkletNode</a> constructor. Entries in this dictionary whose
+              names that correspond to <a>AudioParam</a>s in the class definition of
+              an <a>AudioWorkletProcessor</a> are used to initialize the parameter
+              values upon the creation of a node.
+            </p>
+            <dl title="dictionary AudioWorkletNodeOptions" class="idl">
+              <dt>
+                unsigned long numberOfInputs
+              </dt>
+              <dd>
+                Similar to <a>numberOfInputs</a> in <a>AudioNode</a>.
+              </dd>
+              <dt>
+                unsigned long numberOfOutputs
+              </dt>
+              <dd>
+                Similar to <a>numberOfOutputs</a> in <a>AudioNode</a>.
+              </dd>
+              <dt>
+                unsigned long channelCount
```

Comment:
It would be nice to make AudioWorkletNodeOptions derive from AudioNodeChannelOptions/AudioNodeOptions so you don't have to describe these again. (But AudioNodeOptions isn't in the spec yet, but there is a PR for it.)